packages/grafana-ui/src/components/Table/FooterRow.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { ReactNode } from 'react';
+import { ColumnInstance, HeaderGroup } from 'react-table';
+import { selectors } from '@grafana/e2e-selectors';
+import { getTableStyles, TableStyles } from './styles';
+import { useStyles2 } from '../../themes';
+
+export interface FooterRowProps {
+  totalColumnsWidth: number;
+  footerGroups: HeaderGroup[];
+  footer?: ReactNode[];
+  height?: number;
+}
+
+export const FooterRow = (props: FooterRowProps) => {
+  const { totalColumnsWidth, footerGroups, footer, height } = props;
+  const e2eSelectorsTable = selectors.components.Panels.Visualization.Table;
+  const tableStyles = useStyles2(getTableStyles);
+
+  if (!footer) {
+    return null;
+  }
+
+  return (
+    <table
+      style={{
+        position: 'absolute',
+        width: totalColumnsWidth ? `${totalColumnsWidth}px` : '100%',
+        bottom: '0px',
+      }}
+    >
```

Comment:
It's just a nit, but what about using divs with roles to keep things consistent with how the entire Table component is built?

Reply:
That was in a separate pr that is now merged to main. It was a div following the way it was done initially, though @hugohaggmark asked for more semantic html.  I didn't want to change everything so I only changed the footer.  Using divs with roles seems a bit better for consistency.  @hugohaggmark?

Reply:
Sure, I have no strong opinions as long as we follow a11y guidelines.